packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1438,17 +1458,48 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
             this.client.rebasePosition(end, sequenceNumber, localSeq);
 
         const intervalId = properties?.[reservedIntervalIdKey];
+        const localInterval = this.localCollection.getIntervalById(intervalId);
+
         const rebased: ISerializedInterval = {
             start: startRebased,
             end: endRebased,
             intervalType,
             sequenceNumber: this.client?.getCurrentSeq() ?? 0,
             properties,
         };
+
         if (opName === "change" && (this.hasPendingChangeStart(intervalId) || this.hasPendingChangeEnd(intervalId))) {
             this.removePendingChange(serializedInterval);
             this.addPendingChange(intervalId, rebased);
         }
+
+        if (!localInterval) {
```

Comment:
I think it's possible that we reconnect at a time in which the whole string has been deleted, but we also submitted a subsequent op to delete the interval locally. In that case we'd want similar logic to L1485-L1488 here.

```typescript
if (startRebased === -1 || endRebased === -1) {
    if (localInterval) {
        this.localCollection.removeExistingInterval(localInterval)
    }
    return undefined;
}
```

should also add a test for this case, should be a straightforward extension of existing one